rasa/core/actions/action.py

```diff
@@ -1112,6 +1112,33 @@ async def _execute_validation_action(
             event for event in slot_events if event.key not in validated_slot_names
         ]
 
+    def _fails_unique_entity_mapping_check(
+        self,
+        slot_name: Text,
+        mapping: Dict[Text, Any],
+        tracker: "DialogueStateTracker",
+        domain: "Domain",
+    ) -> bool:
+        from rasa.core.actions.forms import FormAction
```

Comment:
Optional: I would import it globally because it's also used by another function in this file.

Reply:
The problem with global importing FormAction that is that it causes a circular import problem so this was the only solution.